src/axes/categoryAxis.ts

```diff
@@ -117,22 +120,101 @@ namespace Plottable.Axes {
     }
 
     protected _getTickValues() {
-      return this.getDownsampleInfo().domain;
+      return (this._scale as Scales.Category).getDownsampleInfo().domain;
     }
 
     /**
-     * Take the scale and drop ticks at regular intervals such that the resultant ticks are all a reasonable minimum
-     * distance apart. Return the resultant ticks to render, as well as the new stepWidth between them.
+     * Gets the current tick mark position value.
+     */
+    public tickMarkPosition(): ITickMarkPosition;
+    /**
+     * Sets  whether tick marks are drawn in the center of category or after it.
      *
-     * @param {Scales.Category} scale - The scale being downsampled. Defaults to this Axis' scale.
-     * @return {DownsampleInfo} an object holding the resultant domain and new stepWidth.
+     * @param {ITickMarkPosition} tickMarkPosition - "center" or "after"
+     * @returns {Category} The calling Category.
```

Comment:
for consistency I suppose we can keep doing this but fyi we don't need the type info `{Typed}` as typedoc can infer stuff on its own.